extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTableWriter.java

```diff
@@ -842,18 +868,30 @@ static class PrimitiveTransfer<C extends WritableChunk<Values>, B extends Buffer
         private final B buffer;
         private final ColumnSource<?> columnSource;
         private final ChunkSource.FillContext context;
+        private boolean hasMoreDataToBuffer;
 
         PrimitiveTransfer(ColumnSource<?> columnSource, C chunk, B buffer, int targetSize) {
             this.columnSource = columnSource;
             this.chunk = chunk;
             this.buffer = buffer;
             context = columnSource.makeFillContext(targetSize);
+            hasMoreDataToBuffer = false;
```

Comment:
We typically only initialize non-default values.  `hasMoreDataToBuffer` is `false` by default.